src/vector/geometry.rs

```diff
@@ -358,9 +354,7 @@ impl Eq for Geometry {}
 
 pub fn geometry_type_to_name(ty: OGRwkbGeometryType::Type) -> String {
     let rv = unsafe { gdal_sys::OGRGeometryTypeToName(ty) };
```

Comment:
We don't need to free `rv` here as it's essentially a static str, right?

Reply:
> internal human readable string, or NULL on failure.

(but it's never NULL in the current implementation, it returns something like "Unrecognized: XXXX" in a static buffer)